tests/Validation/Collector/EmptyConfigurableRectorCollector/ConfigurableArrayMissingTest.php

```diff
@@ -16,8 +17,9 @@ final class ConfigurableArrayMissingTest extends AbstractTestCase
 
     protected function setUp(): void
     {
-        $this->bootFromConfigFiles([__DIR__ . '/config/configurable_array_missing.php']);
-        $this->emptyConfigurableRectorCollector = $this->getService(EmptyConfigurableRectorCollector::class);
+        $rectorKernel = new RectorKernel();
```

Comment:
changed this tests to mimic what was already done in `FnMatchPathNormalizerTest`.

with this code it is obvious that these tests require the `ContainerBuilder`. thats a special thing only very few tests need.